Build/libHttpClient.141.Android.Java/app/src/main/java/com/xbox/httpclient/HttpClientRequest.java

```diff
@@ -33,16 +35,17 @@ public void setHttpUrl(String url) {
     }
 
     @SuppressWarnings("unused")
-    public void setHttpMethodAndBody(String method, String contentType, byte[] body) {
-        if (body == null || body.length == 0) {
+    public void setHttpMethodAndBody(String method, long call, String contentType, long contentLength) {
+        RequestBody requestBody = null;
+        if (contentLength == 0) {
             if ("POST".equals(method) || "PUT".equals(method)) {
-                this.requestBuilder = this.requestBuilder.method(method, RequestBody.create(NO_BODY, null));
-            } else {
-                this.requestBuilder = this.requestBuilder.method(method, null);
+                MediaType mediaType = (contentType != null ? MediaType.parse(contentType) : null);
+                requestBody = RequestBody.create(mediaType, NO_BODY);
```

Comment:
Related to comment below, if you upgrade to OkHttp version 4.9.1 (I believe the one we upgraded to), the arguments to this method should be reversed as they've deprecated this one.